src/NuGet/BuildNuGets.csx

```diff
@@ -126,6 +129,24 @@ var PreReleaseOnlyPackages = new HashSet<string>
     "Microsoft.VisualStudio.LanguageServices.Next",
 };
 
+// The assets for these packages are not produced when building on Unix
+// and we don't want to attempt to package them when building packages.
+var PackagesNotBuiltOnCore = new HashSet<string>
+{
+     "Microsoft.CodeAnalysis.CSharp.Features",
```

Comment:
I generated this list by just including what failed due to missing build outputs.  You can tell me if it looks sensible or not.